techniques/html/H2.html

```diff
@@ -2,109 +2,100 @@
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/link-purpose-in-context">達成基準 2.4.4: リンクの目的 (コンテキスト内)</a> (参考)
                </li>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/link-purpose-link-only">Success Criterion 2.4.9: Link Purpose (Link Only)</a> (Advisory)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/link-purpose-link-only">達成基準 2.4.9: リンクの目的 (リンクのみ)</a> (参考)
                </li>
             </ul>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>This objective of this technique is to provide both text and iconic representations
-               of links without making the web page more confusing or difficult for keyboard users
-               or assistive technology users. Since different users finding text and icons more usable,
-               providing both can improve the accessibility of the link.
+            <h2>解説</h2>
+            <div class="note">
+               <div role="heading" class="note-title marker" aria-level="3">訳注</div>
+               <div>
+                  <p>WAIC では H2 に関する<a href="https://waic.jp/docs/as/">アクセシビリティ サポーテッド（AS）情報</a>を提供している。2020 年 3 月版の <a href="https://waic.jp/docs/as/info/202003/techs/H2.html">アクセシビリティ サポーテッド（AS）情報: H2</a> も参照されたい。
+                  </p>
+               </div>
+            </div>
+
+            <p>この達成方法の目的は、テキストとアイコンの両方でリンクを提供する際、キーボード利用者や支援技術の利用者にとって混乱や困難を招くような ウェブページを作らないようにすることである。様々な利用者がテキストとアイコンを使いやすいと見出していることから、両方を提供することでリンクのアクセシビリティを向上させることができる。
             </p>
-            <p>Many links have both a text and iconic representation adjacent to each other, but
-               rendered in separate <code class="el">a</code> elements. Visually they appear to be a single link, but many users encounter them
-               as adjacent identical links. For a keyboard user, it is tedious to navigate through
-               redundant links. For users of assistive technologies, it can be confusing to encounter
-               successive identical links. When the text alternative for the icon is a duplicate
-               of the link text, it is repetitive as screen readers read the description twice.
+            <p>よくあるリンクとして、テキストとアイコンの両方が互いに隣接しながらも、別々の <code class="el">a</code> 要素としてレンダリングされているものがある。視覚的にはそれらは単一のリンクのように見えるが、それらを同様のリンクが隣接しているものとして受け取る利用者も多い。キーボード利用者の場合、冗長なリンクを通過していくのは面倒である。支援技術の利用者にとって、連続する同様のリンクに遭遇すると混乱する可能性がある。アイコンのテキストによる代替がリンクテキストの複製である場合、スクリーンリーダーは読み上げを 2 度繰り返すことになる。
             </p>
-            <p>If the author omitted alternative text from the link image, it would fail <a href="https://www.w3.org/TR/WCAG21/#text-equiv-all">Success Criterion 1.1.1</a> because the
-               text alternative would not serve the same purpose as the graphical link.
+            <p>制作者がリンク画像の代替テキストを省略した場合、テキストによる代替が視覚的なリンクと同じ目的を果たさないため、<a href="https://www.w3.org/TR/WCAG21/#text-equiv-all">達成基準 1.1.1</a> を満たせなくなる。
```

Comment:
WCAG 2.1日本語訳に置き換えておきます

```suggestion
            <p>制作者がリンク画像の代替テキストを省略した場合、テキストによる代替が視覚的なリンクと同じ目的を果たさないため、<a href="https://waic.jp/docs/WCAG21/#non-text-content">達成基準 1.1.1</a> を満たせなくなる。
```